component/vshn_postgres.jsonnet

```diff
@@ -330,6 +452,25 @@ local composition =
             // add backup configuration
           ],
         },
+        // {
```

Comment:
You can probably remove the unused code block.

Reply:
ok